github/pulls.go

```diff
@@ -251,12 +251,14 @@ type PullRequestMergeResult struct {
 
 // PullRequestOptions lets you define how a pull request will be merged.
 type PullRequestOptions struct {
-	Squash bool
+	Squash      bool
```

Comment:
It looks to me like `Squash` is no longer supported and can be removed..
https://developer.github.com/changes/2016-04-01-squash-api-preview/ seems to confirm, although it is not explicit.